packages/pointer_interceptor/pointer_interceptor_ios/example/ios/RunnerUITests/RunnerUITests.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+
+final class RunnerUITests: XCTestCase {
+
+  override func setUp() {
+        continueAfterFailure = false
+      }
+
+      func testPointerInterceptorBlocksGesturesFromFlutter() {
+        let app = XCUIApplication()
+        app.launch()
+
+        let fabInitial = app.buttons["Initial"]
+        if (!(fabInitial.waitForExistence(timeout: 30))){
+          print(app.debugDescription);
+          XCTFail("Could not find Flutter button to click on")
+          return
+        }
+
+        fabInitial.tap();
+
+        let fabAfter = app.buttons["Tapped"]
+        if (!(fabAfter.waitForExistence(timeout: 30))){
+          print(app.debugDescription);
+          XCTFail("Flutter button did not change on tap")
+          return
+        }
+
+        let dummyButton = app.staticTexts["Not Clicked"]
```

Comment:
PlatformView and FlutterWidget receive the touch event on different threads. There could be a race condition in this test, it is safer to wait before checking the PlatformView text. I think 3 seconds is good enough. 
(Waiting an arbitrary time is not good in test, if you can think of another way for testing this, it would be better)